main/config.go

```diff
@@ -251,6 +253,7 @@ func (c *Config) setDefaultURLs() error {
 		return fmt.Errorf("invalid UBIRCH backend environment: \"%s\"", c.Env)
```

Comment:
Please note that there are more environments now, there is dev, demo, prod, kdo, and uae-prod, with potentially more on the way.

Reply:
Will make a proper ticket for this on monday.